backend/read_env.py

```diff
@@ -117,6 +123,19 @@ def get_argument_from_env() -> tuple[str, list, list, dict, str, str]:
         cmake_args.append("-DENABLE_PYTORCH=OFF")
         pt_version = None
 
+    if os.environ.get("DP_ENABLE_PADDLE", "0") == "1":
+        pd_install_dir, _ = find_paddle()
+        pd_version = get_pd_version(pd_install_dir)
+        cmake_args.extend(
+            [
+                "-DENABLE_PADDLE=ON",
+                f"-DCMAKE_PREFIX_PATH={pd_install_dir}",
```

Comment:
conflict with line 119

Reply:
I am not sure if paddle really needs it, which depends on how paddle is found in the cmake files.

Reply:
Thanks for pointing out that mistake, `CMAKE_PREFIX_PATH` indeed is not a required variable for Paddle C++ interface installation, it works after removing this.